Src/Base/AMReX.cpp

```diff
@@ -693,7 +693,7 @@ amrex::Finalize (amrex::AMReX* pamrex)
     BL_TINY_PROFILE_FINALIZE();
     BL_PROFILE_FINALIZE();
 
-#ifdef AMREX_USE_CUDA
+#if defined(AMREX_USE_CUDA) || defined(AMREX_USE_HIP)
```

Comment:
```suggestion
#if defined(AMREX_USE_GPU)
```
Let's change this to `AMREX_USE_GPU`.